<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessShow</class>
 <widget class="QWidget" name="ChessShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>1048</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>331</width>
     <height>451</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget_select" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_ai_first">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>ai先手</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_aibehind">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>ai后手</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_NetWar" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>131</width>
     <height>231</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>95</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>显示聊天对话框</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_findteam">
       <property name="text">
        <string>寻找队伍</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>放弃寻找</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>140</y>
     <width>84</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>双人对战</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>ai对战</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>网络对战</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>返回登录</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="text">
       <string>悔棋</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_SelfWon">
        <property name="text">
         <string>我赢了</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_TimeOut">
        <property name="text">
         <string>超时</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>192</x>
     <y>770</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>770</y>
     <width>79</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>鼠标点击的点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_show">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>770</y>
     <width>62</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>770</y>
     <width>31</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>状态:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stable">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>780</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>454</x>
     <y>743</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>提示状态:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>517</x>
     <y>743</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>454</x>
     <y>770</y>
     <width>130</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>网络对战你所执的棋子</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_node">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>760</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>　</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
